src/backend/server.js

```diff
@@ -12,11 +12,10 @@ connectDB();
 
 // Middleware
 app.use(express.json());
-app.use(cors());
```

Comment:
@Sam559I Can I please keep this cors middleware? I need this to work with Frontend integration

Reply:
hey, i'll update this and the  userController